docs/sources/operators-guide/mimir-runbooks/_index.md

```diff
@@ -831,12 +836,14 @@ This alert is fired when the multi-tenant alertmanager has been unable to check
 
 When the alertmanager loads its configuration on start up, when it polls for config changes or when there is a ring change it must check the ring to see if the tenant is still owned on this shard. To prevent one error from causing the loading of all configurations to fail we assume that on error the tenant is NOT owned for this shard. If checking the ring continues to fail then some tenants might not be assigned an alertmanager and might not be able to receive notifications for their alerts.
 
-The metric for this alert is cortex_alertmanager_ring_check_errors_total.
+The metric for this alert is `cortex_alertmanager_ring_check_errors_total`.
 
 How to **investigate**:
 
 Look at the error message that is logged and attempt to understand what is causing the failure. In most cases the error will be encountered when attempting to read from the ring, which can fail if there is an issue with in-use backend implementation.
```

Comment:
We can convert to an unordered list, like in other runbooks, now that we have 2 items.